ReactiveCocoa/NSObject+RACDescription.m

```diff
@@ -24,7 +24,11 @@ - (NSString *)rac_description {
 @implementation NSValue (RACDescription)
 
 - (NSString *)rac_description {
-	return self.description;
+    if (getenv("RAC_DEBUG_SIGNAL_NAMES") != NULL) {
```

Comment:
I don't think we really need the check for this case, since `NSValue`s are usually just primitives.